db/re/item_db_equip.yml

```
@@ -159728,6 +160400,92 @@ Body:
          bonus2 bAddRace,RC_All,5;
       if (.@r>=11)
          bonus2 bAddSize,Size_All,5;
+  - Id: 400487
+    AegisName: Up_Fancy_Flower_IL
+    Name: Opened Illusion Fancy Flower
+    Type: Armor
+    Weight: 100
+    Defense: 18
+    Slots: 1
+    Locations:
+      Head_Top: true
+    ArmorLevel: 1
+    EquipLevelMin: 150
+    Refineable: true
+    View: 4
+    Script: |
+      .@r = getrefine();
+      bonus bMatk,5+(9*(.@r/2));
+      bonus bMatkRate,5;
+      if (.@r>=7)
+         bonus bVariableCastrate,-5;
+      if (.@r>=9)
+         bonus2 bMagicAtkEle,Ele_All,5;
+      if (.@r>=11)
+         bonus2 bMagicAddSize,Size_All,5;
+  - Id: 400480
+    AegisName: aegis_400480
+    Name: Costume Cat Straw Hat
+    Type: Armor
+    Locations:
+      Costume_Head_Top: true
+    ArmorLevel: 1
+    View: 2413
+  - Id: 400481
+    AegisName: aegis_400481
+    Name: The Brandishing
+    Type: Armor
+    Weight: 500
+    Defense: 10
+    Slots: 1
+    Locations:
+      Head_Top: true
+    ArmorLevel: 1
+    EquipLevelMin: 200
+    Refineable: true
+    View: 2415
+    Script: |
+      bonus bCrt,3+getrefine();
+      bonus2 bSubRace,RC_Player_Doram,13;
+      bonus2 bSubRace,RC_Player_Human,13;
+      bonus2 bSubClass,Class_Boss,20;
+      bonus2 bSubClass,Class_Normal,20;
+      bonus bAtkRate,40;
+      bonus2 bSkillAtk,"DK_SERVANT_W_PHANTOM",BaseLevel/10;
+      bonus2 bSkillAtk,"DK_SERVANT_W_DEMOL",BaseLevel/10;
+      bonus2 bSkillAtk,"DK_SERVANTWEAPON",BaseLevel/10;
```

With `bSkillAtk` the skill should always be `DK_SERVANTWEAPON_ATK` according to the data extracted from `EquipmentProperties.lub`. `DK_SERVANTWEAPON` is still used with other bonuses.
```suggestion
      bonus2 bSkillAtk,"DK_SERVANTWEAPON_ATK",BaseLevel/10;
```

---

Oh okay, Thanks !